modules/pel/peltool/pel_values.py

```diff
@@ -194,6 +194,21 @@
     'Symptom':       7}
 
 
+"""
+The possible Action Flags values with the bit numbering needed by bitset.
+"""
+actionFlagsBits = {
```

Comment:
Please use [ActionFlagsValues](https://github.com/ibm-openbmc/openpower-pel-parsers/blob/peltool-testing/modules/pel/peltool/pel_types.py#L10)

Reply:
I should add actionFlagsBits values to [ActionFlagsValues](https://github.com/ibm-openbmc/openpower-pel-parsers/blob/peltool-testing/modules/pel/peltool/pel_types.py#L10) and use this class instead?

Reply:
Yes, we already have an enum class for the action flags, correct? 

The purpose of `pel_values.py` is primarily to print text that helps the user understand PEL data when using peltool to parse it.

Reply:
Okay!